rbac/management/querysets.py

```diff
@@ -95,8 +98,8 @@ def get_role_queryset(request):
     if not res_list:
         return Role.objects.none()
     if '*' in res_list:
-        return Role.objects.annotate(policyCount=Count('policies', distinct=True))
-    return Role.objects.filter(uuid__in=res_list).annotate(policyCount=Count('policies', distinct=True))
+        return base_query
+    return base_query.filter(uuid__in=res_list)
```

Comment:
LGTM overall, but as described in the note section, the "filter" would make a new queryset. Should we use prefetch object instead? https://docs.djangoproject.com/en/1.7/ref/models/querysets/#prefetch-related

Reply:
@astrozzc based on our conversation, it looks like you're correct, however since this is isolated to a case for RBAC on RBAC where a user would have explicit permissions for a subset of roles, I don't think we need to be concerned with this not adding performance improvements to this instance. Thanks for the detailed find!

https://stackoverflow.com/questions/12973929/why-does-djangos-prefetch-related-only-work-with-all-and-not-filter/12974801#12974801